microsoft/testsuites/mdatp/mdatp.py

```diff
@@ -0,0 +1,57 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+import pathlib
+
+from lisa import Node, SkippedException, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.operating_system import Linux
+from lisa.sut_orchestrator import AZURE
+from lisa.testsuite import simple_requirement
+
+
+@TestSuiteMetadata(
+    area="mdatp",
+    category="functional",
+    description="""
+        Tests to check the behavior of the virtual cdrom device in VMs.
```

Comment:
it's inconsistent with test case description. Is it testing the cdrom or defender? What's the "defender"? Is it "Microsoft Defender"?

Reply:
Ah thanks, will fix